.env.example

```
@@ -25,9 +27,13 @@ LOGIN_REDIRECT_URL=
 LOGOUT_REDIRECT_URL=
 CORS_ALLOWED_ORIGINS=
 ACCOUNT_DEFAULT_HTTP_PROTOCOL=
-DRF_THROTTLE_LOGIN=
 ACCOUNT_EMAIL_VERIFICATION=
 MAILGUN_API_KEY=
 MAILGUN_DOMAIN=
 EMAIL_BACKEND=
 DEFAULT_FROM_EMAIL=
+
+DRF_THROTTLE_USER=2000/min
+DRF_THROTTLE_ENDPOINT_ACTOR=500/min
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DRF_THROTTLE_ENDPOINT_ACTOR key should go before the DRF_THROTTLE_USER key

(UnorderedKey)